### ISSUE TITLE
Lightsail and Bitnami Wordpress - A number of site crashing/freezing frequently. 

### ISSUE THREAD

**Issue by JayFEWD** [bitnami/vms#248]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

c0a93e10-d23f-4298-82ac-7fdda76788b6

### bndiagnostic output

===== Begin of bndiagnostic tool output =====

 
    ? Apache: Found possible issues
    ? Mariadb: Found possible issues
    ✓ Connectivity: No issues found
    ✓ Wordpress: No issues found
    ? Resources: Found possible issues
    ✓ Processes: No issues found
    ✓ Php: No issues found
 

 
[Apache]

Found recent error or warning messages in the Apache error log.
```
[Mon Aug 15 03:15:55.833539 2022] [mpm_event:error] [pid 1796:tid 
140407529937792] AH03490: scoreboard is full, not at MaxRequestWorkers.Increase 
ServerLimit.
 [Mon Aug 15 03:15:56.834660 2022] [mpm_event:error] [pid 1796:tid 
140407529937792] AH03490: scoreboard is full, not at MaxRequestWorkers.Increase 
Press [Enter] to continue:
ServerLimit.
 [Mon Aug 15 03:21:23.115095 2022] [proxy_fcgi:error] [pid 1736:tid 
140217948280576] [client **ip_address**:27274] AH01071: Got error 'PHP message: 
PHP Notice: Unknown: file created in the system's temporary directory in Unknown 
on line 0', referer: https://sixty4.net/how-did-we-decide-to-market-sports-bras/

```
Please check the following guide to troubleshoot server issues:
 
https://docs.bitnami.com/general/apps/wordpress/troubleshooting/debug-erro
rs-apache/

[Mariadb]

Found recent error messages in the MariaDB error log:
```
2022-08-14 23:23:49 23910 [Warning] Aborted connection 23910 to db: 
'bitnami_wordpress' user: 'bn_wordpress' host: 'localhost' (Got an error writing 
communication packets)

```
Please check the following guide to troubleshoot MariaDB issues:
 
Press [Enter] to continue:
https://docs.bitnami.com/aws/apps/wordpress/troubleshooting/debug-errors-m
ariadb/

[Resources]

Your instance has little available RAM memory.
```
 total used free shared buff/cache available Mem: 481 292 8 54 180 123 Swap: 634 
228 406
```
You could try to increase your instance's memory. Please check your cloud 
provider's documentation for more information.
 

===== End of bndiagnostic tool output =====

### bndiagnostic was not useful. Could you please tell us why?

I'm sure its helpful, diagnostic isn't my strong point yet!

### Describe your issue as much as you can

Hi Bitnami

I have been using Lightsail and Bitnami Wordpress for many years across 30+ websites and love the setup. 

However lately I have been experiencing a number of Wordpress crashes across a handful of unrelates sites, more and more frequently. 

This current issue is on a site I migrated to a new VM in the last 2 weeks in hope of resolving the problem. This site has low traffic and has previously run without issue for years, now it's crashing daily even on the new VM.

I'd love some guidance and suggestion on what could be causing the issues on a new server so I can resolve this.
(Will tackle  1 server at a time. More to come!)

Thanks

**Comment by mdhont** [comment]
It seems the machine is low on RAM, I recommend using a slightly more powerful instance or try reducing the number of plugins, as some can be very demanding on resources.

**Comment by JayFEWD** [comment]
Hi Thanks for your reply. I will up the ram but don't see what has changed in the last year. Why all of a sudden is RAM an issue when it's been running fine for the previous year?

**Comment by ax0n-pr1me** [comment]
Same issue - possibly related to #225 and #247

**Comment by mdhont** [comment]
> . I will up the ram but don't see what has changed in the last year. Why all of a sudden is RAM an issue when it's been running fine for the previous year?

Please note that newer versions of the application and/or plugins can be more demanding on resources than their previous versions.

**Comment by JayFEWD** [comment]
> > . I will up the ram but don't see what has changed in the last year. Why all of a sudden is RAM an issue when it's been running fine for the previous year?
> 
> Please note that newer versions of the application and/or plugins can be more demanding on resources than their previous versions.

Indeed, good point. Ive had this server running for a day now with half the plugins running. Turning 1 on each hour and watching metrics, it may be as simple as a new plugin version pushing it over. Ill update if I find the issue.

**Comment by JayFEWD** [comment]
As a follow up to this issue for anyone else scratching their head over instance crashes on low traffic VPS instances.
I did remove some Wordpress plugins no longer needed, moved the login page from wp-admin, I increased the swap space on this instance and also introduced some bot management through Cloudflare. So far so good, It's gone from daily crashes to now running 2 weeks without issue. A bigger server instance may not always the answer. Happy to close this for now :)
